charts/civil-service/values.yaml

```diff
@@ -54,6 +54,7 @@ java:
     FEIGN_CLIENT_CONFIG_REMOTEREPOSITORYSERVICE_URL: http://camunda-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/engine-rest/
     SPRING_CLOUD_BOOTSTRAP_ENABLED: true
     RD_PROFESSIONAL_API_URL: http://rd-professional-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    RD_COMMONDATA_API_URL: http://rd-commondata-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

Comment:
Make sure you add this variable in civil service charts  for other repos otherwise it will fail all other repos when it is merged

Reply:
While creating this variable I was following the example of the one above it, RD_PROFESSIONAL_API_URL, and it only exists in this one place as well. Sorry, but I'm still fairly new to all this, so I'm not exactly sure if I know what you mean.

Reply:
Added